src/taxonomy/system-defined-badge/SystemDefinedBadge.scss

```diff
@@ -0,0 +1,7 @@
+.system-defined-badge {
+  font-size: 12px;
```

Comment:
[I think this is equivalent to x-small.](https://paragon-openedx.netlify.app/foundations/typography/
)